### PR TITLE
Add a warning that setting --port flag to Kubelet breaks "kubectl log…

### PR DESCRIPTION
**Pull request by davidopp**
…s." Temporary fix for #9325.

### CODE REVIEW

**Comment by davidopp** [comment]
@dchen1107 @lavalamp

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -186,10 +186,10 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.FileCheckFrequency, "file-check-frequency", s.FileCheckFrequency, "Duration between checking config files for new data")
 	fs.DurationVar(&s.HTTPCheckFrequency, "http-check-frequency", s.HTTPCheckFrequency, "Duration between checking http for new data")
 	fs.StringVar(&s.ManifestURL, "manifest-url", s.ManifestURL, "URL for accessing the container manifest")
-	fs.BoolVar(&s.EnableServer, "enable-server", s.EnableServer, "Enable the info server")
-	fs.Var(&s.Address, "address", "The IP address for the info server to serve on (set to 0.0.0.0 for all interfaces)")
-	fs.UintVar(&s.Port, "port", s.Port, "The port for the info server to serve on")
-	fs.UintVar(&s.ReadOnlyPort, "read-only-port", s.ReadOnlyPort, "The read-only port for the info server to serve on (set to 0 to disable)")
+	fs.BoolVar(&s.EnableServer, "enable-server", s.EnableServer, "Enable the Kubelet")
```
Comment (lavalamp) [inline cmd/kubelet/app/server.go]: ```Kubelet's server```?

**Comment by davidopp** [comment]
Done, PTAL.

**Comment by lavalamp** [comment]
LGTM

**Comment by dchen1107** [comment]
Please squash commits into one, then LGTM

**Comment by davidopp** [comment]
Squashed.

**Pull request merged**